Formula/swiftlint.rb

```diff
@@ -14,6 +14,7 @@ class Swiftlint < Formula
   end
 
   depends_on xcode: ["11.4", :build]
+  depends_on :macos
```

Comment:
If we are able to update to Swift 5.4 (#73984) and successfully build on Linux, we should also be able to try building `swiftlint` on Linux.

So, may want to add a comment to revisit.

Reply:
I added a comment for `swiftlint` suggesting we try to build it on Linux once Swift is available.  From what I can see, `swimat` is an Xcode plugin so it will always be macOS only, and `swiftformat` and `swiftgen` only seem to support building with Xcode.  It will be interesting to see how many other formulae which depend on Swift can actually be built on Linux when Swift is bottled.